dotcom-rendering/src/components/TextBlockComponent.tsx

```diff
@@ -158,7 +159,8 @@ const sanitiserOptions: IOptions = {
 };
 
 const styles = (format: ArticleFormat) => css`
-	margin-bottom: 14px;
+	margin-block: ${space[3]}px;
+	writing-mode: horizontal-tb;
```

Comment:
I’m not sure we’re quite ready yet for [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values/Margins_borders_padding), and once we are we should apply them consistently, otherwise this is at risk of becoming more confusing.